src/python/string/punctuation.js

```diff
@@ -0,0 +1,8 @@
+// Original by: Clari (https://github.com/Clari-7744)
+module.exports = function punctuation () { // eslint-disable-line camelcase
```

Comment:
the `// eslint-disable-line camelcase` is not needed, the function name is a single word, so camelCasing rule doesn't apply here.

Reply:
ok i was basing it off another file, i wasn't sure

Reply:
moved